src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -46,7 +46,7 @@ public void addingComponents() {
   public void savingTemplate() {
     TemplatePage templatePage = new InfoboxBuilderPage()
         .openExisting("InfoboxBuilderSavingTemplate")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "InfoboxBuilderSavingTemplate" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)